test/e2e/next-test-lib/next-test-utils.js

```diff
@@ -2,7 +2,7 @@
 import spawn from 'cross-spawn'
 import { existsSync, readFileSync, unlinkSync, writeFileSync } from 'fs'
 import { writeFile } from 'fs-extra'
-import { fetch as undiciFetch } from 'undici'
+import { fetch as undiciFetch } from 'next/dist/compiled/undici'
```

Comment:
I guess we never figured out why the `undici` dep was no longer included?

Reply:
It happened here https://github.com/netlify/next-runtime/commit/c295bc23bd769e0d946a40d75ee7340f4d5cdf9d
it got removed from package-lock but it looks like it was not set as a dep in the package.json.

Reply:
Once we move to Node.js 18, `fetch` is native, so this import won't be required.